tests/cases/unittest.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+class UnitTestTranslation(unittest.TestCase):
+    def testSequences(self):
+        x = [1, 2]
+        z = x  # Added
+
+        self.assertEqual(z, [1, 2])
+
+        x = [1, 2, 3]
+        y = x
+        self.assertTrue(x is y)
+
+if __name__ =="__main__":
+    unittest.main()
```

Comment:
when I run this locally, the test case fails with

```py
Traceback (most recent call last):
  File "/home/jayvdb/python/py2many/tests/test_cli.py", line 226, in test_generated
    self.assertTrue(expected_output, "Test cases must print something")
  File "/usr/lib64/python3.9/unittest/case.py", line 688, in assertTrue
    raise self.failureException(msg)
AssertionError: b'' is not true : Test cases must print something
```

This restriction is in place because `print` is available in all the transpilers.

Solution is to replace this line with
```py
if __name__ =="__main__":
    print("Running tests")
    unittest.main()
```